specification/web/resource-manager/readme.md

```diff
@@ -88,6 +88,20 @@ directive:
   - suppress: XMS_EXAMPLE_NOTFOUND_ERROR
     from: Provider.json
     reason: Retroactive addition of missing examples out of scope at this time.
+  - from: ContainerApps.json
+    where: $.paths
+    transform: >
+      for (var path in $)
+      {
+          delete $[path];
```

Comment:
I don't think we can just remove it with directive in the SDK side to by pass the api layer breaking change review ? we should remove the corresponding swagger files and should get approval from API breaking change board. @mikekistler @JeffreyRichter

Reply:
same concern for Go SDK.